server/chat-plugins/username-prefixes.ts

```diff
@@ -126,3 +126,11 @@ export const commands: Chat.ChatCommands = {
 		);
 	},
 };
+
+export const handlers: Chat.Handlers = {
+	onConfigRefresh() {
+		// ensure that battle prefixes configured via the chat plugin are not overwritten
+		// by battle prefixes manually specified in config.js
```

Comment:
What battle prefixes are manually specified in config.js? If they're related, maybe it should be stored in the same place the chat plugin's own data is?